src/Core/Commit.js

```diff
@@ -196,6 +183,27 @@ function commitUnmountElementComponent (element, parent, host, children) {
 	unmountComponentInstance(element)
 }
 
+/**
+ * @param {Element} element
+ * @param {Element} parent
+ */
+function commitUnmountElementPromise (element, parent) {
+	element.cache.then(function () {
+		commitOwnerRemove(element, parent)
+	})
+}
+
+/**
+ * @param {Element} element
+ * @param {Element} parent
+ */
+function commitUnmountElementSiblings (element, parent) {
+	if (!parent.active)
+		if (parent.id < SharedElementSnapshot)
+			for (var children = parent.children, sibling = children.next; element !== sibling; sibling = sibling.next)
```

Comment:
This loop stops once it hits `element`, instead it should merely skip `element`

Reply:
Can you expand on that? I first had this as a loop traversing nodes from element going backward to the head of the LinkedList then modeled it to start from head of the LinkedList to the current. Note this only happens when an error happens in the child of a fragment midway its creation, and this logic serves to undo previously created siblings before the problem element that might have been already created.

Reply:
I see, that would explain `!parent.active` (which also handles async child render)